src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,42 @@
+import styled, { css } from "styled-components";
+
+// poniżej próba tak dla sprawdzenia, czy mogę w ogóle zmieniać coś w button, wydaje mi się, że nie bo to jest jakoś dziedziczone tu z bootstrapa, czy dobrze myślę ?
+const StyledButton = styled.button`
+  padding: 40 px;
+  font-size: 40 px;
+  line-height: 1.5;
+  border-radius: 10 px;
+  color: yellow;
+  background-color: violet;
+`;
```

Comment:
Tu masz podstawowe ustawienia, jeśli chcesz coś modyfikować to wew. backtików ``` ` ` ``` musisz to robić.
Zobacz tutaj: https://styled-components.com/docs/basics#adapting-based-on-props